src/server/helpers/mailer.js

```diff
@@ -0,0 +1,14 @@
+import config from '../../../config/config.json';
```

Comment:
See how we import and deal with the configuration here: https://github.com/bookbrainz/bookbrainz-site/blob/master/src/server/app.js#L32

Reply:
i thougt mail configs must be at the global level , that way wont it be easy to setup the project for newbies ?

Reply:
okay i get it